_proto/hapi/services/tiller.proto

```diff
@@ -61,7 +61,7 @@ service ReleaseService {
 	}
 
 	// InstallRelease requests installation of a chart as a new release.
-	rpc InstallRelease(InstallReleaseRequest) returns (InstallReleaseResponse) {
+	rpc InstallRelease(InstallReleaseRequest) returns (stream InstallReleaseResponse) {
```

Comment:
returning a streamed response instead of a regular response is what will break existing clients.